server/proxy/bindings.go

```diff
@@ -72,6 +73,7 @@ func (p *Proxy) GetBindingsForApp(sessionID, actingUserID string, cc *apps.Conte
 	appCC := *cc
 	appCC.AppID = appID
 	appCC.BotAccessToken = app.BotAccessToken
+	appCC.Locale = utils.GetLocale(p.conf.MattermostAPI(), p.conf.MattermostConfig().Config(), actingUserID)
```

Comment:
That is a costly call. Doing that on every bindings call might be a big burden. I'm wondering, if we could deffer getting the locale to the `ExpandedContext` which will contain the user anyway (if the app requested it)

Reply:
If we don't expand the user, then we cannot localise the bindings? Or do you mean taking the local from the user if expanded, and if not, do it this way?

Reply:
I mean relying on the app to request an expanded binding, if it needs it. Most apps won't localize (for now) and hence won't slow down the bindings requests.